core/vm/evm.go

```diff
@@ -541,13 +541,12 @@ func (evm *EVM) initNewContract(contract *Contract, address common.Address) ([]b
 			return ret, ErrCodeStoreOutOfGas
 		}
 	} else {
-		consumed, wanted := evm.AccessEvents.CodeChunksRangeGas(address, 0, uint64(len(ret)), uint64(len(ret)), true, contract.Gas)
-		contract.UseGas(consumed, evm.Config.Tracer, tracing.GasChangeWitnessCodeChunk)
-		if len(ret) > 0 && (consumed < wanted) {
```

Comment:
the `len(ret) > 0` is unnecessary, as the gas cost for empty code is always zero.

Reply:
this seems to come from a rebase, I agree this is now useless